aperturedb/cli/configure.py

```diff
@@ -0,0 +1,193 @@
+import json
+import os
+from json import JSONEncoder
+from pathlib import Path
+
+import typer
+from typing_extensions import Annotated
+
+from aperturedb.cli.console import console
+from aperturedb.Configuration import Configuration
+
+
+class ObjEncoder(JSONEncoder):
```

Comment:
Would be nice to have a explanation of this.

Reply:
I will put the comment.